### ISSUE TITLE
Kessler physics using inconsistent reference pressures

### ISSUE THREAD

**Issue by nusbaume** [ESCOMP/CAM#802]
### What happened?

While going through the use of CCPP standard names in our physics code, I saw that Kessler is being given a "reference" pressure that is the US standard atmosphere sea level pressure (i.e. 1013.25 hPa) to convert from the provided Exner function.  However, the Exner function itself is calculated using a reference pressure of 1000 hPa (which is also used in Klemp et al. referenced in the Kessler code).  Thus when values are calculated using the Exner function in Kessler there is an unphysical ~1.013 offset that occurs.

### What are the steps to reproduce the bug?

Look at `kessler_cam_init` in `physics/simple/kessler_cam.F90`, `kessler_init` and `kessler_run` in `atmos_phys/kessler/kessler.F90`, and `calc_exner_run` in `atmos_phys/utilities/state_converters.F90`.

### What CAM tag were you using?

cam6_3_109

### What machine were you running CAM on?

Other (please explain below)

### What compiler were you using?

Other (please specify below)

### Will you be addressing this bug yourself?

Yes

### Extra info

I will fix this issue when I bring in an updated version of the `atmospheric_physics` external (issue #752)

**Comment by adamrher** [comment]
Nice catch, Jesse. Should we also use a shared variable in computing exner in ```calc_exner_run```:
```
    do i=1,nz
      exner(:ncol,i) = (pmid(:ncol,i)/1.e5_kind_phys)**(rair/cpair)
    end do
```
since the reference pressure is hard coded here.

**Comment by nusbaume** [comment]
@adamrher Yes, good call!  I've gone ahead and made a new issue in the atmospheric_physics repo to make sure we address this, which can be found here:

https://github.com/NCAR/atmospheric_physics/issues/61

I'll hopefully implement the change right before I update the externals here in CAM.
